contracts/utils/package.json

```diff
@@ -0,0 +1,90 @@
+{
+    "private": true,
```

Comment:
Is this private for now? Until we're all done?

Reply:
Yes